zeebe-cluster/values.yaml

```diff
@@ -16,6 +16,10 @@ ioThreadCount: "2"
 pvcSize: "10Gi"
 pvcAccessModes: [ "ReadWriteOnce" ]
 
+gateway:
```

Comment:
I think we should rename it to standaloneGateway to make it more clear. Furthermore we should again add a enabled flag, which is per default false. If this is true we can have logic which sets the embedded to disabled with help of the negate operator for example. https://helm.sh/docs/chart_template_guide/functions_and_pipelines/#operators-are-functions

Reply:
:+1: 

So we add capability to switch between embedded and standalone then?

Reply:
I don't think that is actually needed in the helm charts.. we should thing of Helm charts as the most typical way of deployment for users to work with Zeebe.. and I see that embedded gateway in the brokers to be just one option.. if someone ask for it we should look into see if it is doable..